QUEST_LV_0200.tsv

```diff
@@ -9230,51 +9230,51 @@ QUEST_LV_0200_20151001_009233	The space is gradually collapsing.{nl}I don't know
-QUEST_LV_0200_20151001_009272	Dionysus is not someone we can face against.{nl}Even Vakarine in her old days would not be able to do anything.
-QUEST_LV_0200_20151001_009273	The power of the evening star key is accumulating in the magical suppressor.{nl}We are all ready and we have enough time.
-QUEST_LV_0200_20151001_009274	We are going to detach the power from Dionysus from now on.{nl}If we fail, it would bring the same result as if tens of Demon Lords are on the loose.{nl}
-QUEST_LV_0200_20151001_009275	We need to approach it with caution. {nl}Let me know when you are ready.
-QUEST_LV_0200_20151001_009276	Hauberk ran towards 4th prison which is protected by Daiva. {nl}First, you should return to the Vakarine. I will look after Dionysus.
-QUEST_LV_0200_20151001_009277	Savior. {nl}You are now ready to know the truth.{nl}The power which I gave to Dionysus is called the chain of the returning.{nl}
-QUEST_LV_0200_20151001_009278	The chain of the returning is the wonder which connects every boundaries or return...{nl}Hauberk wants to use the chain of the returning to connect his spirits.{nl}
-QUEST_LV_0200_20151001_009279	But, Hauberk too is the key to close the space of the metastasis.{nl}The reason why I left him like that was to store the power so that the key functions well.{nl} 
-QUEST_LV_0200_20151001_009280	Hauberk hasn't collected his spirit in full yet so it wouldn't be hard to face against.{nl}Catch him again and close the space of the metastasis with the chain of the returning.{nl}
-QUEST_LV_0200_20151001_009281	The 4th district which he ran away to is like a maze. {nl}Kupole Daiva and you would be able to follow him.
+QUEST_LV_0200_20151001_009237	$So you're the Revelator Vakarine talked about. {nl}It is great that you aren't late.{nl}
+QUEST_LV_0200_20151001_009238	$The Demon Lord Nuaele's power is weakening the Prison at an alarming rate.{nl}However, we Kupoles have no way to stop Nuaele.{nl}
+QUEST_LV_0200_20151001_009239	$And Kupole Aldona told me that she couldn't just wait for the Revelator...{nl}Without the time for persuasion, she went after Nuaele.
+QUEST_LV_0200_20151001_009240	$Please stop the demons that are coming here through the gaps.{nl}If you could stop them before they join Nuaele, it will be great help to the safety of Aldona.
+QUEST_LV_0200_20151001_009241	$Goddess.{nl}I hope nothing occurs to Aldona...
+QUEST_LV_0200_20151001_009242	$With her forces diminished, Nuaele's power should be weakened.{nl}Before anything happens to Aldona, I will tell you what to do next.
+QUEST_LV_0200_20151001_009243	$Nuaele is difficult to face head-on with only us.{nl}But Nuaele wasn't always that strong.{nl}
+QUEST_LV_0200_20151001_009244	$Nuaele got strong only after obtaining pieces of Hauberk's marks. {nl}Hauberk's subordinates who lost their master started to follow her since she had the pieces of the mark.{nl}
```

Comment:
that means the item you collect has an incorrect name. I was matching the dialogue with the item name